starter-code/css/main.css

```diff
@@ -1,8 +1,90 @@
-/*
-Colors:
+/*Colors:
 
 Text: 1A1A1A
 Green: #00B172
 White: #FFF
-
 */
+
+
+.container {
+  border: 5px solid red;
+}
+
+ nav {
+    border: 5px solid yellow;
```

Comment:
setting a border to your boxes can be useful to plan your layout indeed! Dont forget to erase it when you re done or you will have very colorful websites 😀